Project.toml

```diff
@@ -12,11 +12,21 @@ Clustering = "aaaa29a8-35af-508c-8bc3-b662a17a0fe5"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
-JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
 JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
 julia = "^1.0"
+
+[extras]
+Cbc = "9961bab8-2fa3-5c5a-9d89-47fab24efd76"
+JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+[targets]
+test = ["Pkg", "Test", "StatsBase", "Cbc", "Random", "JLD2"]
```

Comment:
Great use

Reply:
So apparently if you use Pkg.test, you need [extras] and [targets], and if you use `julia runtests.jl`, you need a Project.toml file in the test folder. Somehow Pkg.test fails the tests for us whereas the second option works for all tests (no idea why), I'll open an issue with example in julia discourse on that...

Reply:
Btw, I just scrolled through the documentation, it seems that this will be streamlined in julia 1.2